fdbserver/SimulatedCluster.actor.cpp

```diff
@@ -863,9 +863,12 @@ void SimulationConfig::generateNormalConfig(int minimumReplication, int minimumR
 		case 2:
 			set_config("log_version:=3");  // 6.1
 			break;
+		case 3:
```

Comment:
Similar to previous comment, this whole switch block perhaps can instead be `if (logVersion >= MIN_SUPPORTED && logVersion < MAX_SUPPORTED) { set_config(format("log_version=%d", logVersion)); }` so that this doesn't need to change each time version is bumped.